tanjun/commands.py

```diff
@@ -2198,8 +2195,9 @@ def find_command(self, content: str, /) -> collections.Iterable[tuple[str, abc.M
             return
 
         for command in self._commands:
-            if (name := utilities.match_prefix_names(content, command.names)) is not None:
-                yield name, command
+            # `name_` is `name`, but mypy doesn't allow redefining types (from `str` to `Optional[str]`)
```

Comment:
I should probably remove this comment given I didn't leave it for anything else, hmmm